roc/version.go

```diff
@@ -72,3 +73,40 @@ func parseVersion(s string) SemanticVersion {
 
 	return v
 }
+
+func stringifyVersion(sv SemanticVersion) string {
+	return fmt.Sprintf("%d.%d.%d", sv.Major, sv.Minor, sv.Patch)
+}
+
+func checkVersionCompatibility(nativeVersion, bindingsVersion SemanticVersion) error {
```

Comment:
nit: we can replace two args with one arg `Versions`. Also maybe rename to just `checkCompatibility`?

I think `checkCompatibility(versions)` is not ambiguous and reads well.